connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -298,6 +309,15 @@ message StreamingQueryCommandResult {
     // Logical and physical plans as string
     string result = 1;
   }
+
+  message ExceptionResult {
+    // Exception message as string
+    optional string exception_message = 1;
+  }
+
+  message AwaitTerminationResult {
+    optional bool terminated = 1;
```

Comment:
We don't need separate message if this is just one field. We can directly use bool.

Reply:
will fix in a separate PR with other fields